.github/workflows/quick.yaml

```
@@ -1,8 +1,15 @@
-name: GitHub CI
+# The jobs below are executed on every PR branch push (among other
+# events), quickly and cheaply detecting many common PR problems.
+#
+# Anubis configuration must be kept in sync with the total number of
+# tests executed for "auto" branch commits, including tests in other
+# GitHub workflow file(s).
+
+name: PR commit
```

This workflow also tests master and staged commits. None of them are really "PR commit"s. There was an old change request about that somewhere.

```suggestion

name: GH quick tests
```

or

```suggestion

name: Quick tests
```

or

```suggestion
```


The first and last option would need the corresponding slow.yaml update, but there is a dedicated change request for that file.

The last one is my favorite.